ChensGradiusMod/Extensions.cs

```diff
@@ -130,7 +130,7 @@ public static void InitializeDroneModelComponents(this GameObject droneModel, Ch
             characterModel.BuildRendererInfos(droneModel);
             characterModel.autoPopulateLightInfos = true;
             characterModel.invisibilityCount = 0;
-            characterModel.temporaryOverlays = new List<TemporaryOverlay>();
+            characterModel.temporaryOverlays = new List<TemporaryOverlayInstance>(); //I used this bootleg aah method in Titans of the Rift so hopefully it works here too..?
```

Comment:
We'd prefer if this is tested.

Reply:
![image](https://github.com/user-attachments/assets/51205faa-661d-44c6-86bc-43e92043a550)

according to the "Upgrading mods to SOTS" thread under the #wiki channel on official modding discord, this is the intended way to do it as Direseeker, Arsonist, Lee Hyperreal mods already utilizes this.

Reply:
We still want visual confirmation about it. Are you able to setup the locally and see of Chill Drone would appear as normal? If it does, then it works.

Reply:
![image](https://github.com/user-attachments/assets/7ab4860d-d340-4893-a808-343f83369d29)
![image](https://github.com/user-attachments/assets/b571d6b2-9eb7-474f-a7b2-dcb63a50188e)

no errors in the console either, so I think its a success